tests/data/setting_data.py

```diff
@@ -5,11 +5,20 @@
 from scripts.utils.chain_model import Chain
 
 network_file_path = os.getenv('CHAINS_JSON_PATH', "chains/v10/chains.json")
-skipped_networks = ['Edgeware', 'Ethereum']
```

Comment:
lets also add here Bifrost Kusama and Kintsugi temporarily until we will decide what to do with DOT on Kusama parachains

Reply:
It will skip it at all, from all tests, such sorting need to do closer to particular test. I will add that logic